extensions/dot-new/src/index.tsx

```diff
@@ -12,6 +12,12 @@ export function getFilteredDataset(query?: string): DataSet {
   return !query ? dataset : fuse.search(query, { limit: 10 }).map((result) => result.item);
 }
 
+export const getFaviconUrl = (domain: string | undefined) => {
+  if (domain) {
+    return `https://www.google.com/s2/favicons?sz=64&domain=${encodeURI(domain)}`;
+  }
+};
+
```

Comment:
Something I just did locally was adding fallback images to this function in the Safari extension :D 

Here is the code snippet:

```
export const getFaviconUrl = (domain: string | undefined) => {
  if (!domain) {
    return Icon.Globe;
  }

  return {
    source: `https://www.google.com/s2/favicons?sz=64&domain=${encodeURI(domain)}`,
    fallback: Icon.Globe,
  };
};
```

Reply:
Oh, thanks for sharing that @thomaspaulmann. Here's the update: https://github.com/raycast/extensions/pull/1254/commits/edb1acd5d04ac231a0cbd5d95c0c834aed5595af.